digits/model/tasks/torch_train.py

```diff
@@ -112,7 +112,7 @@ def create_mean_file(self):
         filename = os.path.join(self.job_dir, constants.MEAN_FILE_IMAGE)
         # don't recreate file if it already exists
         if not os.path.exists(filename):
```

Comment:
Notice here that if `filename` exists, it will in this case always use `constants.MEAN_FILE_IMAGE` and thus the original `mean.jpg` will override anything you might manually supply yourselves. I am now wondering why the constant is used here in the first place and not just `.get_mean_file()`. I think it's because torch uses the jpg and caffe the proto